From 69d296df3d49ea14027519d351e35353d89ed745 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 11 May 2022 15:40:02 +0100
Subject: [PATCH] overlays: Remove other leading zeroes from node
 addresses

Newer versions of dtc warn about node addresses with leading zeroes.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts | 2 +-
 arch/arm/boot/dts/overlays/imx219-overlay.dts      | 2 +-
 arch/arm/boot/dts/overlays/imx258-overlay.dts      | 2 +-
 arch/arm/boot/dts/overlays/ov5647-overlay.dts      | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

--- a/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
@@ -37,7 +37,7 @@
 			#size-cells = <0>;
 			status = "okay";
 
-			sgtl5000@0a {
+			sgtl5000@a {
 				#sound-dai-cells = <0>;
 				compatible = "fsl,sgtl5000";
 				reg = <0x0a>;
--- a/arch/arm/boot/dts/overlays/imx219-overlay.dts
+++ b/arch/arm/boot/dts/overlays/imx219-overlay.dts
@@ -64,7 +64,7 @@
 				};
 			};
 
-			vcm: ad5398@0c {
+			vcm: ad5398@c {
 				compatible = "adi,ad5398";
 				reg = <0x0c>;
 				status = "disabled";
--- a/arch/arm/boot/dts/overlays/imx258-overlay.dts
+++ b/arch/arm/boot/dts/overlays/imx258-overlay.dts
@@ -74,7 +74,7 @@
 				};
 			};
 
-			vcm: ad5398@0c {
+			vcm: ad5398@c {
 				compatible = "adi,ad5398";
 				reg = <0x0c>;
 				status = "disabled";
--- a/arch/arm/boot/dts/overlays/ov5647-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ov5647-overlay.dts
@@ -39,7 +39,7 @@
 				};
 			};
 
-			vcm: ad5398@0c {
+			vcm: ad5398@c {
 				compatible = "adi,ad5398";
 				reg = <0x0c>;
 				status = "disabled";
